connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1610,13 +1613,15 @@ ble_error_t Gap::update_ll_address_resolution_setting()
 #if BLE_ROLE_BROADCASTER
 uint8_t Gap::getMaxAdvertisingSetNumber()
 {
+    tr_info("getting max. advertising set number");
```

Comment:
instead print the trace when the set number is known

Reply:
I'm worried we call this function too often to log this every time

Reply:
I share that concern with you @paul-szczepanek-arm . My rule of thumb would be to not add log to getter excepts if it does lazy initialization (or start unexpected operations 😱). In that case, log the path of the operation but do not log the _getting_ part.